### ISSUE TITLE
Wobbly headers when used along `infinite_listview`

### ISSUE THREAD

**Issue by ghost** [fluttercommunity/flutter_sticky_headers#27]
When using this library along with [`infinite_listview`](https://pub.dev/packages/infinite_listview), some headers of the negative indexes get flicky. It happens until the index 0 is visible, then it stops.

You should be able to see some blank spaces between some headers and the AppBar:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/47678313/65888674-4d0f3880-e376-11e9-8dab-2f53412f9534.gif)

The code used in the example above:

```dart
import 'package:flutter/material.dart';
import 'package:infinite_listview/infinite_listview.dart';
import 'package:sticky_headers/sticky_headers.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Example(),
      theme: ThemeData(primarySwatch: Colors.blueGrey),
    );
  }
}

class Example extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Example'),
      ),
      body: InfiniteListView.builder(
        itemBuilder: (_, int index) {
          return StickyHeader(
            header: header(context),
            content: ListTile(
              title: Text('Item #$index'),
              subtitle: Text('Description'),
            ),
          );
        },
      ),
    );
  }

  Container header(BuildContext context) {
    return Container(
      padding: const EdgeInsets.all(16),
      width: double.infinity,
      color: Colors.blueGrey[700],
      child: Text(
        'Header',
        style: Theme.of(context).textTheme.title.copyWith(color: Colors.white),
      ),
    );
  }
}
```

**Comment by TatsuUkraine** [comment]
Had same issue https://github.com/fluttercommunity/flutter_sticky_headers/issues/19, since I didn't get any response, I wrote own implementation of sticky headers that doesn't have this issue

**Comment by hugocastelani** [comment]
@TatsuUkraine Would you mind sharing it with us?

**Comment by TatsuUkraine** [comment]
@hugocbpassos https://pub.dev/packages/sticky_infinite_list

**Comment by phatwerk** [comment]
Having the same issue. It happens after scrollTo/jumpTo is called of ScrollablePositionedList (Google).

**Comment by slightfoot** [comment]
This should be fixed as-of v0.3.0.
